tests/e2e/test_splunk_addon.py

```diff
@@ -174,11 +201,13 @@ def empty_method():
 
     # run pytest with the following cmd args
     result = testdir.runpytest(
-        "--splunk-type=docker",
+        "--splunk-type=kubernetes",
+        "--splunk-web-scheme=http",
+        "--splunk-version={0}".format(request.config.getoption("splunk_version")),
         "-v",
         "-m splunk_searchtime_fields",
         "--search-interval=4",
-        "--search-retry=4",
+        "--search-retry=10",
```

Comment:
What is the reason of changing the search retry value in the tests?

Reply:
https://splunk.atlassian.net/browse/SPL-218937